target/linux/ipq806x/files-4.9/arch/arm/boot/dts/qcom-ipq4019-jalapeno.dts

```diff
@@ -0,0 +1,351 @@
+				qcom,poll_required = <1>;
+				qcom,poll_required_dynamic = <1>;
+				qcom,phy_mdio_addr = <4>;
+				vlan_tag = <2 0x20>;
+			};
+		};
+
+		mdio@90000 {
+			status = "ok";
+		};
+
+		ess-switch@c000000 {
+			status = "ok";
+			switch_lan_bmp = <0x10>; /* lan port bitmap */
+		};
+
+		ess-psgmii@98000 {
+			status = "ok";
+		};
+
+		spi_0: spi@78b5000 {
+			pinctrl-0 = <&spi_0_pins>;
+			pinctrl-names = "default";
+			cs-gpios = <&tlmm 54 0>, <&tlmm 59 0>;
+			status = "ok";
+
+			mx25l25635e@0 {
+				status = "disabled";
+			};
+
+			m25p80@0 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				reg = <0>;
+				compatible = "mx25l1606e", "n25q128a11";
+				linux,modalias = "m25p80", "mx25l1606e", "n25q128a11";
+				spi-max-frequency = <24000000>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					partition0@0 {
+						label = "SBL1";
+						reg = <0x00000000 0x00040000>;
+						read-only;
+					};
+					partition1@40000 {
+						label = "MIBIB";
+						reg = <0x00040000 0x00020000>;
+						read-only;
+					};
+					partition2@60000 {
+						label = "QSEE";
+						reg = <0x00060000 0x00060000>;
+						read-only;
+					};
+					partition3@c0000 {
+						label = "CDT";
+						reg = <0x000c0000 0x00010000>;
+						read-only;
+					};
+					partition4@d0000 {
+						label = "DDRPARAMS";
+						reg = <0x000d0000 0x00010000>;
+						read-only;
+					};
+					partition5@e0000 {
+						label = "APPSBLENV"; /* uboot env*/
+						reg = <0x000e0000 0x00010000>;
+						read-only;
+					};
+					partition5@f0000 {
+						label = "APPSBL"; /* uboot */
+						reg = <0x000f0000 0x00080000>;
+						read-only;
+					};
+					partition5@170000 {
+						label = "ART";
+						reg = <0x00170000 0x00010000>;
+						read-only;
+					};
+				};
+			};
+
+			mt29f@1 {
+				status = "ok";
+				#address-cells = <1>;
+				#size-cells = <0>;
+				compatible = "spinand,mt29f", "w25n01gv";
+				reg = <1>;
+				spi-max-frequency = <24000000>;
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					partition0@0 {
+						label = "ubi";
+						reg = <0x00000000 0x08000000>;
+					};
+				};
+			};
+		};
+
+		counter@4a1000 {
+			compatible = "qcom,qca-gcnt";
+			reg = <0x4a1000 0x4>;
+		};
+		
+		mdio@90000 {
+			status = "okay";
+		};
+		
+		ess-switch@c000000 {
+			status = "ok";
+		};
+
+		serial@78af000 {
+			pinctrl-0 = <&serial_pins>;
+			pinctrl-names = "default";
+			status = "ok";
+		};
+
+		cryptobam: dma@8e04000 {
+			status = "ok";
+		};
+
+		crypto@8e3a000 {
+			status = "ok";
+		};
+
+		watchdog@b017000 {
+			status = "ok";
+		};
+
+		usb3_ss_phy: ssphy@9a000 {
+			status = "ok";
+		};
+
+		usb3_hs_phy: hsphy@a6000 {
+			status = "ok";
+		};
+
+		usb3: usb3@8af8800 {
+			status = "ok";
+		};
+
+		usb2_hs_phy: hsphy@a8000 {
+			status = "ok";
+		};
+
+		usb2: usb2@60f8800 {
+			status = "ok";
+		};
+
+		usb3_ss_phy: ssphy@9a000 {
```

Comment:
You have many nodes duplicated...

Reply:
I reused a lot of 8devices DTS and that is part of it,was not paying attention to it.
Removed 3 copies of USB nodes.